packages/js/data/src/product-attribute-terms/index.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME, WC_PRODUCT_ATTRIBUTE_TERMS_NAMESPACE } from './constants';
+import { createCrudDataStore } from '../crud';
+
+createCrudDataStore( {
+	storeName: STORE_NAME,
+	resourceName: 'ProductAttributeTerm',
+	pluralResourceName: 'ProductAttributeTerms',
+	namespace: WC_PRODUCT_ATTRIBUTE_TERMS_NAMESPACE,
+	urlParameters: [ 'attribute_id' ],
```

Comment:
Same comment as previously, this isn't needed anymore right? since you are parsing it from the `namespace` string?